versions.gradle

```diff
@@ -33,6 +33,8 @@ ext {
       lithoAnnotations    : "com.facebook.litho:litho-annotations:$lithoVersion",
       lithoProcessor      : "com.facebook.litho:litho-processor:$lithoVersion",
 
+      jsr305              : "com.google.code.findbugs:jsr305:3.0.2",
```

Comment:
Shouldn't this come from javax.annotation instead?

Reply:
Looking into https://github.com/eclipse-ee4j/common-annotations-api/ (the new home of `javax.annotation`), there is no `Nullable`. JSR 305 actually never came to fruition. Adding `javax.annotation:javax.annotation-api:1.3.2` therefore does not help. The findbugs implementation is what most others use (Guava, OkHttp). Using `javax.annotation` is nowadays problematic anyway because [it breaks on the JPMS](https://blog.codefx.org/java/jsr-305-java-9/).